### DISCUSSION TITLE
How to navigate the archive content ?

### DISCUSSION THREAD

**Discussion by ghost** [gokcehan/lf#986] [General]
I'm not just talking about extraction. Most of the archives such as zip, are openable & their content is readable.
Is it possible navigate the content of the archives, like most of the file managers do ?

**Comment by ilyagr** [comment]
No, this is not implemented in `lf`.

**Reply by ghost** [reply]
A workaround I had (to just read through internal file structure), is to open archives with neovim. It'll list each file per line.

**Comment by DusanLesan** [comment]
I think closest you can get is by mounting the archive. I use [archivemount](https://github.com/cybernoid/archivemount) to be able to both read and write, but there should be several other solutions that are faster but mount archive as read only. 
This is my implementation of archivemount that mounts archive in the same directory and moves into it:
```
cmd open ${{
	case $(file --mime-type "$f" -b) in
		*/zip) lf -remote "send $id mount_archive" ;;
		*) for f in $fx; do setsid $OPENER $f >/dev/null 2>&1 & done ;;
	esac
}}

cmd mount_archive ${{
	mntdir="$f-archivemount"
	[ ! -d "$mntdir" ] && {
		mkdir "$mntdir"
		archivemount "$f" "$mntdir"
	}
	lf -remote "send $id cd \"$mntdir\""
	lf -remote "send $id reload"
}}
```
